README.md

```diff
@@ -186,14 +188,21 @@ the repository is located at `/opt/thingengine` and the local state directory is
 
 ### Step 5: the web frontend
 
+First set `SECRET_KEY` in your environment:
+
+```sh
+SECRET_KEY=secret
+```
+`SECRET_KEY` is used to hash the session. Choose a secure secret to prevent session hijaking.
```

Comment:
Nice, I had forgotten about this.
By the way, there are a couple more secret keys that are needed:
- JWT_SECRET_KEY: used for OAuth access tokens, same format as SECRET_KEY
- AES_SECRET_KEY: used for 2-factor keys, must be exactly 32 hex characters

Reply:
Where exactly should these keys be implemented?

Reply:
Environment variables, same as `SECRET_KEY`